server/services/workAndSkillsPageService.ts

```diff
@@ -161,20 +163,27 @@ export default class WorkAndSkillsPageService {
     return learnerProfiles
   }
 
-  private async getLearnerEducation(
-    prisonerNumber: string,
-    curiousApiClient: CuriousApiClient,
-  ): Promise<GovSummaryItem[]> {
-    const learnerEducation: LearnerEducation = await curiousApiClient.getLearnerEducation(prisonerNumber)
-    const coursesAndQualifications: GovSummaryItem[] = []
-    learnerEducation?.content?.forEach(content => {
-      const item = {
-        key: { text: content.courseName },
-        value: { text: `Planned end date on ${formatDate(content.learningPlannedEndDate, 'long')}` },
-      }
-      coursesAndQualifications.push(item)
-    })
-    return coursesAndQualifications
+  private async getLearnerEducation(prisonerNumber: string, token: string): Promise<GovSummaryItem[]> {
+    const inPrisonCourseRecords: InPrisonCourseRecords = await this.curiousService.getPrisonerInPrisonCourses(
+      prisonerNumber,
+      token,
+    )
+
+    /* InPrisonCourseRecords are already grouped by course status. At the moment the current UI does not differentiate by status
+       so all courses are merged into a single array.
+       TODO - RR-630 will refactor this and display and sort courses by status
```

Comment:
As per the TODO comment, we will change this in the subsequent PRs that we will do in this epic